app/main.py

```diff
@@ -1,13 +1,18 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
-    pass
+    return {"line": error['line_number'],
+            "column": error['column_number'],
+            "message": error['text'],
+            "name": error['code'],
+            "source": "flake8"
+            }
```

Comment:
Bad code style. You have alone closing curly brace and not alone the opening one. Rewrite.